executor/index_merge_reader.go

```diff
@@ -621,14 +620,10 @@ func (w *partialIndexWorker) extractTaskHandles(ctx context.Context, chk *chunk.
 	return handles, retChk, nil
 }
 
-func (w *partialIndexWorker) buildTableTask(handles []int64, retChk *chunk.Chunk) *lookupTableTask {
-	var indexOrder map[int64]int
-	var duplicatedIndexOrder map[int64]int
+func (w *partialIndexWorker) buildTableTask(handles []kv.Handle, retChk *chunk.Chunk) *lookupTableTask {
 	task := &lookupTableTask{
-		handles:              handles,
-		indexOrder:           indexOrder,
-		duplicatedIndexOrder: duplicatedIndexOrder,
```

Comment:
Should initialize these two fields ?

Reply:
The original code just declared but not initialized the two maps.
And I checked that IndexMergeReader doesn't care about the order of the rows, so the two maps are not used.